### PR TITLE
Reworking of ACLs of entire RBM system in program modules

### PR DESCRIPTION
**Pull request by bwrsandman**
- Bumping all versions to 1.10.
- Base Group will serve as basic read rights to most objects used in RBM.
- Adding Employee group to take over functional group that was Base
  - This group can create Results and modify them in draft state, they can also validate
- Adding Coordinator Group who's role is to validate indicator values, available with the `program_evaluation` module.
  - capture d ecran de 2015-01-20 10 56 29
- Evaluation group is a sub group of employee but with access to evaluation menu, available with the `program_evaluation` module
- Groups Director, Admin and DPE lose their cascading inheriting rights and now just inherit Base's rights
  - Their roles are only to validate states of results between `validated` and `openned`
- Manager inherits everyone's rights and gains the ability to
  - Edit the configuration
  - Edit `program.result.level`
  - Delete objects

### CODE REVIEW

**Comment by bwrsandman** [comment]
One pep8 fix waiting on #19

**Comment by llacroix** [comment]
:+1:

File: program/program_result.py [hunk 1]
```diff
@@ -211,20 +199,25 @@ def hide_elem(elem, invisible="true"):
             modifiers['invisible'] = invisible
             elem.attrib['modifiers'] = simplejson.dumps(modifiers)
 
+        def ro_elem(elem, readonly="true"):
+            modifiers = simplejson.loads(elem.attrib.get('modifiers', "{}"))
```
Comment (foutoucour) [inline program/program_result.py:213]: docstring please. The method is not easy to read/understand.

File: program_indicator/program_result.py [hunk 2]
```diff
@@ -32,3 +47,25 @@ class program_result(orm.Model):
             'program.result.indicator', 'result_id', 'Result Indicator',
         ),
     }
+
+    def fields_view_get(
+            self, cr, user, view_id=None, view_type='form', context=None,
+            toolbar=False, submenu=False):
+        """
+        Look for the strings "[('state', '!=', 'draft')]"
```
Comment (foutoucour) [inline program_indicator/program_result.py:59]: why do you need this hack?

File: program/program_result.py [hunk 3]
```diff
@@ -198,33 +188,46 @@ def fields_view_get(
...
+        Non-employees cannot edit the form except for changing the state
 
         Hide intervention_id in all but last level results
         Hide pages and fields according to the config parameters of
         the level
         """
         def hide_elem(elem, invisible="true"):
+            """Set field element's invisibility status
+
+            :param lxml.etree.Element elem: View's field
```
Comment (foutoucour) [inline program/program_result.py:200]: That is not compatible with sphinx doc.

If you want to make it compatible please change to

```
:param elem: View's field
:type elem: lxml.etree.Element
```

or more simple:

```
:param elem: lxml.etree.Element, View's field
```

Comment (bwrsandman) [inline program/program_result.py:200]: I'm just following the examples in here http://sphinx-doc.org/domains.html#info-field-lists

That format is used

Comment (foutoucour) [inline program/program_result.py:200]: Oh... never mind, I didn't know this syntax.
Keep rolling then :v:

**Comment by yvaucher** [comment]
:+1:

**Pull request merged**